cms/djangoapps/contentstore/course_info_model.py

```diff
@@ -160,25 +160,13 @@ def _get_index(passed_id=None):
     return 0
 
 
-def _get_html(course_updates_items):
-    """
-    Method to create course_updates_html from course_updates items
-    """
-    list_items = []
-    for update in reversed(course_updates_items):
-        # filter course update items which have status "deleted".
-        if update.get("status") != CourseInfoModule.STATUS_DELETED:
-            list_items.append(u"<article><h2>{date}</h2>{content}</article>".format(**update))
-    return u"<section>{list_items}</section>".format(list_items="".join(list_items))
-
-
 def save_course_update_items(location, course_updates, course_update_items, user=None):
     """
     Save list of course_updates data dictionaries in new field ("course_updates.items")
     and html related to course update in 'data' ("course_updates.data") field.
     """
     course_updates.items = course_update_items
-    course_updates.data = _get_html(course_update_items)
+    course_updates.data = ""
```

Comment:
Because we want to use what is in the `data` field when we render on the off-chance that someone is doing something custom (like XML courses or something else).  Otherwise we render the `items`.  For this to work we need to clear out the `data` field.  This is the incremental approach which would migrate courses over to this method as they introduce new updates, rather than having a big bang data migration to wipe out this field for certain courses